brave/brave-core
# Android syncv1 deprecated infobar

Resolves https://github.com/brave/brave-browser/issues/10416 

## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues/10416 ) for my issue if one did not already exist.
- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on


Here is how it looks
![Screenshot_20200720-225952](https://user-images.githubusercontent.com/24739341/87988898-c173bd80-cae9-11ea-8b12-7458b20a9a2f.png)


## Test Plan:

A. Behavior on fresh install
1. Install a new apk
2. Ensure no infobar displayed

B. Upgrade from sync v1
1. Install version with sync v1 (1.11.98 or earlier)
2. Enable sync v1
3. Install this PR apk
4. Ensure infobar is displayed

C. 
1. Follow step scenario B
2. Restart browser
3. Ensure no infobar displayed

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
